browsers-ng.js

```diff
@@ -1,15 +1,80 @@
 /* eslint-disable */
 /* eslint camelcase: [0] */
 
+// Also Karma's/Webdriver's `browserNames` are case-sensitive
+// Reformat since we made `argv.browsers` lowercase earlier
+// so it plays nice with Karma/Webdriver
+const format = (browser) => {
+  if (browser.includes('headless')) {
+    // `firefoxheadless` -> `firefoxHeadless`
+    browser = browser.replace('headless', 'Headless');
+  }
+
+  // `chrome` -> `Chrome`
```

Comment:
This will also return `FirefoxHeadless`, is that the expectation? I'd expand the comment for clarity
```suggestion
  // 'chrome' => 'Chrome' or 'firefoxHeadless' => 'FirefoxHeadless'
```